nestjs/src/courses/stats/dto/course-stats.dto.ts

```diff
@@ -1,10 +1,16 @@
 import { ApiProperty } from '@nestjs/swagger';
 
 export class CourseStatsDto {
-  constructor(stats: { studentsActiveCount: number; studentsTotalCount: number; studentsWithMentorCount: number }) {
+  constructor(stats: {
+    studentsActiveCount: number;
+    studentsTotalCount: number;
+    studentsWithMentorCount: number;
+    studentsWithCertificateCount: number;
+  }) {
     this.studentsActiveCount = stats.studentsActiveCount;
     this.studentsTotalCount = stats.studentsTotalCount;
     this.studentsWithMentorCount = stats.studentsWithMentorCount;
+    this.studentsWithCertificateCount = stats.studentsWithCertificateCount;
```

Comment:
not blocking, but I would reverse naming here, like `certifiedStudentsCount` `activeStudentsCount` `totalStudents`